app/views/external_works/fetch.js.erb

```diff
@@ -2,14 +2,17 @@
   $j('#external_work_author').val("<%= @external_work.author %>").change();
   $j('#external_work_title').val("<%= @external_work.title %>").change();
   $j('#external_work_summary').val("<%= escape_javascript(@external_work.summary&.html_safe) %>").change();
-  $j('#external_work_summary').html("<%= escape_javascript(@external_work.summary&.html_safe) %>").change();
   $j('#fetched').val("<%= @external_work.id %>");
   $j('#external_work_rating_string').val("<%= @external_work.rating_string %>");
-  var categories = <%= @external_work.category_strings.to_json.html_safe %>;
-  $j('input[name="external_work[category_strings][]"]').each(function() {
```

Comment:
This already work, but we can make it more specific:

```js
$j('input[type="checkbox"][name="external_work[category_strings][]"]').each(function() {
```
to remove a hidden field from the matched elements.

Reply:
I went with a slightly shorter version -- I'm inputs with an `id` that starts with `external_work_category_strings_`, which the hidden field doesn't have.